aeppl/joint_logprob.py

```diff
@@ -81,7 +86,7 @@ def joint_logprob(
 
     # We add `ShapeFeature` because it will get rid of references to the old
     # `RandomVariable`s that have been lifted; otherwise, it will be difficult
-    # to give good warnings when an unaccounted for `RandomVairiable` is
+    # to give good warnings when an unaccounted for `Antimicrobial` is
```

Comment:
What's that?

Reply:
Ha, must've been in my kill ring; no idea how that word would've gotten there, though.